gcovr/tests/cmake_oos_ninja/Makefile

```diff
@@ -0,0 +1,33 @@
+
+all:
+	@echo "CC:   ${CC} (origin: $(origin CC))"
+	@echo "CXX:  ${CXX} (origin: $(origin CXX))"
+	@echo "GCOV: ${GCOV} (origin: $(origin GCOV))"
+	mkdir -p build/build
+	cd build/build && cmake -DCMAKE_TOOLCHAIN_FILE=../../Toolchain.cmake -G "Ninja" -S ../..
```

Comment:
@Spacetown why did you change this? any special reason?

Reply:
Because in the other CMake test it is done the same way.

Reply:
ok, thats the "old" way

Reply:
You can change it back., this was a try to fix the test. Pleas keep the direct ninja call to be sure that ninja is used.

Reply:
I'm fine with it as it is now.
fyi
The whole thing about the `cmake --build` call is that its using the correct buildtool defined by the generator.
Additional arguments can also be passed after another `--` so with appending `-- -v` you could also achieve verbose builds

Reply:
> I'm fine with it as it is now.
> fyi
> The whole thing about the `cmake --build` call is that its using the correct buildtool defined by the generator.
> Additional arguments can also be passed after another `--` so with appending `-- -v` you could also achieve verbose builds

I was searching for something like this but if I run `cmake -h` there isn't a hint to the lonesome dash. For me it will be fine to use the new style in this test.